tests/Functional/App/cases/Basic/config.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:content-api="http://libero.pub/schema/content-api-bundle"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd http://libero.pub/schema/content-api-bundle http://libero.pub/schema/content-api-bundle/content-api-bundle-1.0.xsd">
```

Comment:
Symfony seems to have some magic to rewrite the URIs to the local filesystem when validating (https://github.com/symfony/symfony/blob/dd432c4f1e5d003a7d39f3d827b7d2bb95403e88/src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php#L591-L608). Means the tests work, but people editing won't be able to access it. Looks like it actually works without it being their too, so let's drop it for now (since it's not hosted).